src/api/spec/features/beta/webui/maintenance_workflow_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'browser_helper'
+
+RSpec.describe 'MaintenanceWorkflow', js: true, vcr: true do
+  let(:admin_user) { create(:admin_user) }
+  let(:maintenance_project) do
+    create(:maintenance_project,
+           name: 'MaintenanceProject',
+           title: 'official maintenance space',
+           target_project: [update_project, another_update_project],
+           maintainer: maintenance_coord_user)
+  end
+  let(:user) { create(:confirmed_user, :with_home, login: 'tom') }
+  let(:maintenance_coord_user) { create(:confirmed_user, :with_home, login: 'maintenance_coord') }
+  let(:project) { create(:project_with_repository, name: 'openSUSE:11.4') }
+  let(:another_project) { create(:project_with_repository, name: 'openSUSE:11.5') }
+  let(:branched_project) { create(:project_with_repository, name: 'home:tom:branches:Update') }
+  let(:package) { create(:package_with_file, project: project, name: 'cacti') }
+  let!(:branched_package) { create(:package_with_file, project: branched_project, name: 'cacti.openSUSE_11.4_Update', file_name: 'DUMMY_FILE', file_content: 'boo#12345') }
+  let(:another_package) { create(:package_with_file, project: another_project, name: 'cacti') }
+  let!(:another_branched_package) { create(:package_with_file, project: branched_project, name: 'cacti.openSUSE_11.5_Update', file_name: 'DUMMY_FILE', file_content: 'boo#12345') }
+  let(:update_project) { create(:update_project, maintained_project: project, name: "#{project}:Update") }
+  let(:another_update_project) { create(:update_project, maintained_project: another_project, name: "#{another_project}:Update") }
+  let(:bs_request_without_patchinfo) do
+    create(:bs_request_with_maintenance_incident_actions, source_project_name: branched_project.name, source_package_names: [package.name], target_project_name: maintenance_project.name,
+                                                          target_releaseproject_names: [update_project.name, another_update_project.name])
+  end
```

Comment:
I think you can simplify the setup of `update_project` and `another_update_project` with a block of code too. Like

```ruby
let(:another_update_project) do
  create(:project_with_repository, name: 'openSUSE:11.5')
  create(:package_with_file, project: another_project, name: 'cacti')
  create(:update_project, maintained_project: another_project, name: "#{another_project}:Update")
end
```